gamestonk_terminal/README.md

```diff
@@ -242,16 +243,19 @@ Command|Description|Brokers
 
 &nbsp;
 ## Portfolio Optimization [»](portfolio_optimization/README.md)
-command|desciription
+
+Command|Description
 ------|------
-`equal_weight`|Returns equal weights for all selected stocks
-`mkt_cap`| Returns weights that are weighted by market cap
-`div_yield`|Returns weights that are weighted by dividend yield
-`max_sharpe`| Returns portfolio of Efficient Frontier with maximum sharpe ratio
-`min_vol`| Returns portfolio on Efficient Frontier with minimum volatility
-`eff_risk`| Returns portfolio on Efficient Frontier that maximizes returns at a given risk
-`eff_ret`| Returns portfolio on Efficient Frontier that minimizes risk at a given return level
-`show_eff`|Plots random portfolios and shows the Efficient Frontier
+`add`| add ticker to optimize
+`select`| overwrite current tickers with new tickers
+`equal`| equally weighted
+`property`| weight according to selected info property (e.g. marketCap)
+`maxsharpe`| optimizes for maximal Sharpe ratio (a.k.a the tangency portfolio)
+`minvol`| optimizes for minimum volatility
+`maxquadutil`| maximises the quadratic utility, given some risk aversion
```

Comment:
I had intentionally left that out since it is more complicated (does market neutral break the pie chart since there will be negative weights?). But definitely another option

Reply:
Yeah, I was looking at the API class and just saw there was another function so just added it since it was there.

I tested that only 1/2, and saw negative values but nothing broke. Let me investigate this further later on

Reply:
So I finally got around to looking at it and for market neutral, the pie just shows positive weights.  Not sure if its simple enough to have a (-n AND --pie) condition that tells you there are negative weights and suppress the plot

Reply:
Done!